assets/javascripts/discourse/components/full-page-chat.js

```diff
@@ -57,12 +60,14 @@ export default Component.extend({
       return;
     }
 
-    const sidebarScroll = document.querySelector(
-      ".sidebar-container .scroll-wrapper"
-    );
-    if (sidebarScroll) {
-      sidebarScroll.scrollTop = sidebarScroll.scrollHeight;
-    }
+    schedule("afterRender", () => {
+      const sidebarScroll = document.querySelector(
+        ".sidebar-container .scroll-wrapper"
+      );
+      if (sidebarScroll) {
+        sidebarScroll.scrollTop = sidebarScroll.scrollHeight;
+      }
+    });
```

Comment:
This one is called from `didInsertElement`, so maybe there's no need to delay it?

Reply:
yep I'm actually doing few tests on this, agreed it's probably overkill

Reply:
@CvX look at last changes please, I think that should actually be enough